packages/flutter_tools/test/flutter_manifest_test.dart

```diff
@@ -359,6 +359,80 @@ flutter:
       final FlutterManifest flutterManifest = await FlutterManifest.createFromString(manifest);
       expect(flutterManifest.isEmpty, false);
     });
+
+    test('test parsing appVersion major.minor.patch+build', () async {
```

Comment:
Please follow convention on test names: the group name is `FlutterManifest` and the test names should be predicates about that class. E.g. `parses major.minor.patch+build version clause`.

Reply:
done